XCTest.xcodeproj/project.pbxproj

```diff
@@ -255,6 +299,24 @@
 /* End PBXLegacyTarget section */
 
 /* Begin PBXNativeTarget section */
+		19D66F931DE6F57E00C112C0 /* SwiftXCTest-iOS */ = {
```

Comment:
> I'm not sure about Carthage, though I would consider it to be their bug if it doesn't work, since xcodebuild deals with that setup just fine.

Carthage is supporting such unified schemes from September 2015.